ci-operator/step-registry/baremetalds/metallb-e2e/test/baremetalds-metallb-e2e-test-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds metallb e2e test command ************"
+
+case "${CLUSTER_TYPE}" in
+packet)
+    # shellcheck source=/dev/null
+    source "${SHARED_DIR}/packet-conf.sh"
+    # shellcheck source=/dev/null
+    source "${SHARED_DIR}/ds-vars.conf"
```

Comment:
Probably you can remove it, doesn't seem to be really used

Reply:
changed.